src/test/java/BaseTest.java

```diff
@@ -10,4 +29,75 @@ public static void chromeConfigs() {
         }
     }
 
+    public static void clickSubmitBtn() {
+        WebElement loginButton= BaseTest.driver.findElement(By.cssSelector("button[type='submit']"));
+        loginButton.click();
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = BaseTest.driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.click();
+        passwordField.sendKeys(password);
+    }
+
+    public static void provideEmail(String email) {
+        WebElement emailField = BaseTest.driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.click();
+        emailField.sendKeys(email);
+    }
+
+    @AfterMethod
+    public static void tearDownBrowser() {
+        driver.quit();
+    }
+
+    protected static void validation() throws InterruptedException {
+
+        WebElement myPlaylist = driver.findElement(By.xpath("//*[@id=\"playlists\"]/ul/li[3]/a"));
+        myPlaylist.click();
+        Thread.sleep(2000);
+        WebElement songWasAdded = driver.findElement(By.xpath("//tr[@class='song-item']//td[contains(text(),'Epic Song')]"));
+        Assert.assertTrue(songWasAdded.isDisplayed());
+    }
+
+    protected static void abdisPlaylist() throws InterruptedException {
+
+        WebElement myPlaylist = driver.findElement(By.xpath("//section[@id='songsWrapper']//li[5]"));
+        myPlaylist.click();
+        Thread.sleep(2000);
+    }
+
+    protected static void addSongToPlaylistBtn() throws InterruptedException {
+
+        WebElement addingTheSong = driver.findElement(By.xpath("//button[@title='Add selected songs to…']"));
```

Comment:
Try changing the locator to `//button[@class='btn-add-to']`

![image](https://user-images.githubusercontent.com/104761482/197727492-41299e17-9b4d-4d3d-8f85-2e7e3aa71250.png)